Unreal Engine plugin documentation update (#7626)
getsentry/sentry-docs: src/platform-includes/getting-started-primer/unreal.mdx

Unreal Engine SDK builds on top of other Sentry SDKs and extends them with Unreal Engine specific features. It gives developers helpful hints for where and why an error or performance issue might have occurred.

**Features:**

- Native support for automatic crash error tracking for
  - Android by using the [Android SDK](/platforms/android/) to support Java, Kotlin, C and C++
  - iOS by using the [iOS SDK](/platforms/apple/guides/ios/) to support Objective-C, Swift, C and C++
  - Windows (UE 5.2+) and Linux by using the [Native SDK](/platforms/native/) to support C and C++ with minidumps
  - macOS by using the [macOS SDK](/platforms/apple/guides/macos/) to support Objective-C, Swift, C and C++
- Compatible with [Crash Reporter Client](/platforms/unreal/configuration/setup-crashreporter/) provided along with Unreal Engine
- [Release health](/platforms/unreal/configuration/releases/) to keep track of crash free users and sessions

<Alert level="warning">

> [Crash Reporter Client](/platforms/unreal/configuration/setup-crashreporter/) has to be configured in order to capture crashes on Windows automatically with UE 5.1 or older. Starting from UE 5.2 engine provides an API, which allows you to switch between default and third-party crash-handling solutions and thus this step isn't required.

Reviewer: ```suggestion
[Crash Reporter Client](/platforms/unreal/configuration/setup-crashreporter/) has to be configured in order to capture crashes on Windows automatically with UE 5.1 or older. Starting from UE 5.2, the provided API allows you to switch between default and third-party crash-handling solutions, so this step isn't required.
```
